salt/covid19.sls

```diff
@@ -1,5 +1,14 @@
-{% from 'lib.sls' import set_firewall %}
+{% from 'lib.sls' import set_firewall,create_user %}
+
+{% set entry = pillar.python_apps.covid19admin %}
+{% set userdir = '/home/' + entry.user %}
+{% set directory = userdir + '/' + entry.git.target %}
```

Comment:
These variables aren't used, so can be deleted.

```suggestion
{% set userdir = '/home/' + entry.user %}
{% set directory = userdir + '/' + entry.git.target %}
```

Reply:
removed these two lines